### ISSUE TITLE
Start of line or start of string confusion

### ISSUE THREAD

**Issue by PhilReedData** [LibraryCarpentry/lc-data-intro#163]
In [episode 1](https://github.com/LibraryCarpentry/lc-data-intro/blob/gh-pages/_episodes/01-regular-expressions.md), in the section "Learning common regex metacharacters", we introduce the caret anchor to match the start of the line. 

Should we say start of the string instead of line? What happens when there is a multi-line string, does it match the start of the line or the start of the string. This question came up when delivering episode 2, matching `^[Cc]ommuni` from the multi-line code of conduct string.

**Comment by sharilaster** [comment]
@PhilReedData I think the usage here is line because we have not yet introduced the concept of 'string.' Do you want to put together a PR that would explain the idea and integrate it in the lesson, to see how it works out?

Thoughts on this, everyone? @ppival @ccronje @libcce and others?

**Comment by ccronje** [comment]
@PhilReedData @sharilaster this is something that came up for me also in a workshop - I do prefer using the term 'string'. It depends how much explanation is required to make the distinction clear.

**Comment by libcce** [comment]
@sharilaster I'm good w/ whatever @ccronje and @PhilReedData come up with here 😄

**Comment by PhilReedData** [comment]
(I can't edit my fork of this file at the moment while I have a pull request active on another issue. At least I don't think I can.)

The 'Learning common regex metacharacters' section has the following sentences:
> In computation, a ‘string’ is a contiguous sequence of symbols or values. 
> For example, a word, a date, a set of numbers (e.g., a phone number), or an alphanumeric value (e.g., an identifier). 

I suggest we include one more sentence immediately after:
> A string could be any length, ranging from empty (zero characters) to one that spans many lines of text (including line break characters).

That would not add too much text to an already (necessarily) heavy introduction, it doesn't go into (unnecessary) detail about line ending types and characters. 

We might want to also add one more sentence such as:
> The terms 'string' and 'line' are sometimes used interchangeably, even when they are not strictly the same thing.

**Comment by ccronje** [comment]
Thanks @PhilReedData I'm happy to do a PR for this. Re your question, you can keep editing your fork and submitting PRs, so long as it doesn't clash with a PR you're waiting on (e.g. different changes to the same line of the same file), then you'll get a merge conflict, but you can resolve those via the GitHub GUI which is handy :-)
